dumper/app/src/main/java/com/google/edwmigration/dumper/application/dumper/connector/snowflake/SnowflakeLogsConnector.java

```diff
@@ -161,21 +161,14 @@ public void addTasksTo(List<? super Task<?>> out, ConnectorArguments arguments)
         out.add(new DumpMetadataTask(arguments, FORMAT_NAME));
         out.add(new FormatTask(FORMAT_NAME));
 
-        String queryFormat = newQueryFormat(arguments);
-
-        final int daysToExport = arguments.getQueryLogDays(7);
-        if (daysToExport <= 0)
-            throw new MetadataDumperUsageException("At least one day of query logs should be exported; you specified: " + daysToExport);
-        LOG.info("Log entries within the last {} days will be exported in increments of 1 hour.", daysToExport);
```

Comment:
Is this message going away on purpose?

Reply:
It's already logged in [here](https://github.com/google/dwh-migration-tools/blob/858e7988f048024eeb9593adaf5d84a395c2c1e0/dumper/app/src/main/java/com/google/edwmigration/dumper/application/dumper/connector/ZonedIntervalIterable.java#L97)